README.md

```diff
@@ -2,197 +2,230 @@
 
 [![Build Status](https://travis-ci.org/vtex/toolbelt.svg?branch=master)](https://travis-ci.org/vtex/toolbelt) [![npm version](https://img.shields.io/npm/v/vtex.svg?style=flat)](https://www.npmjs.com/package/vtex) [![Greenkeeper badge](https://badges.greenkeeper.io/vtex/toolbelt.svg)](https://greenkeeper.io/)
 
-All you need to start using the VTEX platform.
+The CLI that offers all you need to start using the VTEX IO platform efficiently.
+
+#### Summary
+- [How do we empower you?](#how-do-we-empower-you)
+- [First step: Installing the VTEX Toolbelt](#installing-the-toolbelt-with-yarn)
+- [Developing an app locally](#to-develop-locally-use-vtex-link)
+- [Customize your prompt](#wanna-know-something-cool-you-can-customize-your-prompt)
+- [FAQ](#frequently-asked-questions)
+- [Contributing](#contributing)
+- [Send us feedback](#tell-us-what-you-think-about-toolbelt)
+
+# How do we empower you? 
+
+VTEX Toolbelt is a command-line program that gives you access to all of 
+the features in the VTEX platform and increases your productivity, allowing you to:
+
+- **Manage your apps quickly:** List, install and uninstall available apps in your account;
+- **Work safely on your personal environments:** List, create and remove workspaces;
+- **Easily ship changes to your final user:** Promote a workspace to `master`, making it public in your account URL;
+- **See what you are creating in real time:** Develop new apps in link mode, using automatic synchronization;
+- **Go live from the command line:** Publish apps to the VTEX App Store.
+
+# Getting started: Installing the VTEX Toolbelt
+To use the VTEX Toolbelt, you'll need to install other engines 
+(that also will be useful when developing for VTEX).
+
+#### Node.js to run the tool
+- Install Node.js here: Toolbelt is written in JavaScript and Node.js. 
```

Comment:
Missing link? I thought of adding a little more to the text in this case because of technicalities (I don't know if it's entirely correct to say that some application is written for Node.js, rather it's meant to be run in Node.js). Also toolbelt I thought now that maybe we would want to say that toolbelt is written in Typescript, which gives an idea of a safer code)
```suggestion
- Install Node.js [here](https://nodejs.org/en/download/): Toolbelt is written in Typescript and designed to run on Node.js. 
```
A background on this:
. Javascript the language for writing code for the Web. There are lots of javascript engines, which are applications that take some javascript code and run it - the V8 (which chrome uses), Chakra (used by Microsof's Edge browser), SpiderMonkey (used by Firefox). Note that the examples I mentioned are browsers. At some point a programmer had an idea - Let's use javascript for servers and system applications, not only on browsers - and then Node.js was born. It uses V8 to run the javascript (just like chrome) and it provides a runtime to run javascript outside of the browser (what does it mean? It means that Node.js creates a bridge between the javascript code and the system functionalities - The browser can't interact with the files on your computer for example - Node.js provides this functionality)
. Typescript is a language built on top of javascript and provides type safety for programmers. It means we programmers can write bug-less code much easier, but it doesn't affect the end user, only the programmers (maybe it affects the end user in the way that we ship code that is less likely to have bugs). Even though we write the application in typescript, in the build pipeline this typescript is translated into javascript, and then node.js runs it.